com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/HDRaytracingRecursiveRenderer.cs

```diff
@@ -25,6 +26,20 @@ void InitRecursiveRenderer()
             m_FlagMaskTextureRT = RTHandles.Alloc(Vector2.one, TextureXR.slices, colorFormat: GraphicsFormat.R8_SNorm, dimension: TextureXR.dimension, enableRandomWrite: true, useDynamicScale: true, useMipMap: false, name: "FlagMaskTexture");
         }
 
+        public TextureHandle CreateFlagMaskTexture(RenderGraph renderGraph)
+        {
+            return renderGraph.CreateTexture(new TextureDesc(Vector2.one, true, true)
+            {
+                colorFormat = GraphicsFormat.R8_SNorm,
+                slices = TextureXR.slices ,
```

Comment:
slices, dimension and useDynamicScale are already set by the 2 parameters of the TextureDesc constructor so they are redundant here.